assets/src/edit-story/components/canvas/multiSelectionMovable.js

```diff
@@ -68,6 +69,14 @@ function MultiSelectionMovable({ selectedElements }) {
   useGlobalKeyDownEffect('meta', () => setCanSnap(false), [setCanSnap]);
   useGlobalKeyUpEffect('meta', () => setCanSnap(true), [setCanSnap]);
 
+  // ⇧ key rotates the element 30 degrees at a time
+  useGlobalKeyDownEffect('shift', () => setThrottleRotation(true), [
+    setThrottleRotation,
```

Comment:
I don't think this dep is required. It's guaranteed to be a constant anyway.

Reply:
Done